src/Command/Docker/Build.php

```diff
@@ -233,4 +252,31 @@ public function isEnabled(): bool
     {
         return !$this->environment->isMasterBranch();
     }
+
+    /**
+     * @param array $unsupportedErrorMsg
+     * @param array $emptyVersions
+     * @return string
+     */
+    private function createQuestionMessage(array $unsupportedErrorMsg, array $emptyVersions): string
+    {
+        $questionMessage = '';
+        if ($emptyVersions) {
+            $questionMessage .= sprintf(
+                "Some service versions were not provided:\n" .
+                "Using the versions specified in your configuration files for the following services: %s\n" .
+                "If the service is missing in configuration files it will not be installed.\n",
```

Comment:
"Only services defined in the configuration files will be installed."

Reply:
or "If a service is not defined in the configuration files, it will not be installed."

Reply:
I already changed to the first variant. Do I need to change the message to the second variant?

Reply:
Actually second is better, I'll change